to-do-list.js

```diff
@@ -2,15 +2,27 @@ const input = document.getElementById('text-field');
 
 const createListItem = function() {
   const listItem = document.createElement('li');
+  const listItemSpan = document.createElement('span');
+  listItemSpan.classList.add('uncompleted');
```

Comment:
You do not need the `uncompleted` class I guess. Just add `task-item` class which will be a standard class for each li.